README.md

```diff
@@ -1 +1,3 @@
-# Hello-World
\ No newline at end of file
+Project Name - Hello Microverse
+Description - Hello World html template with red color.
+Built with - HTML and CSS with linters
```

Comment:
- [ ] **[REQUIRED CHANGES]**

- Hey, great fixing the changes required by the previous reviewer, however, you've not add all the necessary headers to your readme file to make it look professional. kindly do the following:
-  Copy and paste the content of this [microverse readme template](https://github.com/microverseinc/readme-template) into your readme file.
- change the `Project Name` to `Hello Microverse`
- Add a little description of the project. like the aim of the project
- Remove the `Live Demo (if available)` for now, except you know how to use the `gh` pages to add live link. You will learn this later in the curriculum
- In the `Builth With` header, Add this:
```

- HTML
- CSS
```
- In the `Getting Started` header, add the following:
```

To get a local copy up and running:

1. Clone this repository or download the zip folder:

**``git clone https://github.com/HtetNaing0814/Hello-World.git``**

1. Navigate to the location of the folder in your machine:

**``you@your-Pc-name:~$ cd <folder>``**

```
- In the `Authors` header, change it to be singular, and the add replace the links to your personal detail in the social media networks..  Here:

```
## Author

👤 **your name**

- Github: [@your nme](your GitHub link)
- Twitter: [@your name](your Twitter link) # if and only if you have one
- LinkedIn: [@your name](your Linkedin link) # if and only if you have one
```
- In the `Contribution header` change the issue link to match the issue link of your project like this 
```
Feel free to check the [issue page](https://github.com/HtetNaing0814/Hello-World/issues)
```

- Update the `Acknowledgement` header by adding the microverse official link:
```
 [Microverse](https://bit.ly/MicroverseTN)
```
- In the `License` header add the content in the `MIT.md` file. 
You either add it as a file or add it as a link to the license, like this:
```
This project is [MIT](https://github.com/microverseinc/readme-template/blob/master/MIT.md
) licensed # as a link

This project is [MIT](./MIT.md) licensed # add a file by adding the `MIT.md` file in the root folder of your `Hello Microverse Project` 

```